crt_portal/cts_forms/templates/landing.html

```diff
@@ -99,7 +99,7 @@ <h1 class="h1__display">{% trans "We uphold the civil rights of <em>all</em> peo
         <div class="tablet:grid-offset-2 tablet:grid-col-8">
           <img src="{% static "img/ic_phone-circle.svg" %}" alt="phone" class="icon">
           <p class="h3__display crt-landing--emergency_header">
-            {% trans 'If you are in danger, contact <a href="tel:911">9-1-1</a>' %}
+            {% trans 'If you are in danger, contact <a href="tel:911">911</a>' %}
```

Comment:
I missed this the first time around but for consistency I believe we should fix `9-1-1` to have no dashes either :point_down: 

```html
              {% trans 'If you or someone else is in immediate danger, <a href="tel:911">please call 9-1-1 or local police.</a>' %}
```

Also the spanish translation needs to be adjusted there too.

Reply:
Other than the above issue, I believe this PR is ready to go!